page/layouts/partials/custom-footer.html

```diff
@@ -1,3 +1,4 @@
+<script src="https://contao.org/files/js/network.min.js"></script>
```

Comment:
Just a nitpick, this could be in the `header.html` as well

Reply:
Yepp @fritzmg: As you like?

Reply:
On https://contao.org, https://docs.contao.org and https://partners.contao.org it is at the end of the `<body>`.

Reply:
Thank you, Franko. Will you include the script at the end of the `body` as Fritz suggested?

Reply:
@netzarbeiter It is already, next to the other scripts, before the closing body tag?